src/storage/mod.rs

```diff
@@ -9,24 +9,25 @@ pub mod memory;
 pub mod temp;
 
 /// Creates a [StorageReader] based on the content length returned from the
-/// [SourceStream](crate::source::SourceStream).
+/// [SourceStream](crate::source::SourceStream).And construct a writable handle.
```

Comment:
Can we update this doc comment slightly?

```
/// Creates a [StorageReader] and [StorageWriter] based on the content length returned from the
/// [SourceStream](crate::source::SourceStream).
/// The reader and writer must track their position in the stream independently.
```

Reply:
Thanks for the new text :)